std/protocols/lookup.asm

```diff
@@ -93,12 +92,12 @@ let lookup: expr, expr[], Fp2<expr>, Fp2<expr>, Constr, expr -> Constr[] = |is_f
         )
     );
 
-    let (acc_1, acc_2) = unpack_ext(acc_ext);
+    let is_first: col = std::well_known::is_first;
 
-    [
-        // First and last acc needs to be 0
-        // (because of wrapping, the acc[0] and acc[N] are the same)
-        is_first * acc_1 = 0,
-        is_first * acc_2 = 0
-    ] + constrain_eq_ext(update_expr, from_base(0))
+    let (acc_1, acc_2) = unpack_ext(acc_ext);
+    // First and last acc needs to be 0
+    // (because of wrapping, the acc[0] and acc[N] are the same)
+    is_first * acc_1 = 0;
+    is_first * acc_2 = 0;
+    constrain_eq_ext(update_expr, from_base(0));
```

Comment:
I am thinking about changing the below function in the same way, but I think it is not possible since `constr` function returns `()` but I need to return `Constr[]` to be able to use `bus_interaction` to implement bus_lookup as bus lookup needs adding two constraints like `bus_send() + bus_receive()` https://github.com/powdr-labs/powdr/blob/e668cdfc6438aec58739732e82f01baf114648aa/std/protocols/bus.asm#L39-L61